_posts/2018-07-25-strimzi-0-5-0-release.md

```diff
@@ -0,0 +1,166 @@
+
+The other way is to provide a reference to a complete log4j configuration stored externally in a ConfigMap.
+
+# Rack awareness
+
+In order to have high-availability, Apache Kafka provides a feature called "rack awareness".
+When your topics are replicated across multiple brokers, it is desirable for those brokers to be in different physical racks.
+If a rack fails, you are sure that other topic replicas are alive, a new leader can be elected and clients can continue to exchange messages. As opposite, if the brokers hosting replicas for a topic are put into the same rack, the partitions are not available anymore on rack failure because all brokers will go down at same time.
+
+The new Strimzi rack awareness feature provides a way to specify a `topologyKey` in order to spread brokers across different nodes.
+The `topologyKey` defines a node label which is used as `broker.rack` for configuring the Kafka brokers accordingly.
+
+```yaml
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    # ...
+    rack:
+      topologyKey: failure-domain.beta.kubernetes.io/zone
+    # ...
+```
+
+When deploying the Kafka cluster in the public Cloud (i.e. Amazon, Azure and GCP), it means spreading brokers across different availability zones which can be compared to the physical racks.
+
+# Affinity and Tolerations
+
+Having Kafka brokers colocated on the same nodes with some other applications running on the same Kubernetes/OpenShift cluster can cause some issues like the _noisy neighbour_ phenomenon.
+This is really a problem when these applications are intensively utilizing the same resource - for example network bandwidth or disk I/O.
+
+Thanks to the support for affinity and tolerations in the 0.5.0 release, it's now possible to fix all the above problems.
+You'll get more information about this new feature in a dedicated blog post.
+
+# Cluster Operator handling RBAC
+
+The Cluster Operator is now in charge to handle RBAC resource so `ServiceAccount`, `RoleBinding` and `ClusterRoleBindings` for Kafka pods and for Topic Operator pods which need permissions for accessing resources (i.e. ConfigMap, Node, ...).
+
+An important thing to notice is that the `ClusterRoleBindings` have the need to specify the namespace where the `ServiceAccount` lives for binding it to a specific `ClusterRole`. The provided example YAML files specify the default `myproject` namespace as the following:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: strimzi-cluster-operator
+  labels:
+    app: strimzi
+subjects:
+  - kind: ServiceAccount
+    name: strimzi-cluster-operator
+    namespace: myproject
+roleRef:
+  kind: ClusterRole
+  name: strimzi-cluster-operator
+  apiGroup: rbac.authorization.k8s.io
+```
+
+In order to deploy the cluster properly, remember to change the namespace accordingly with what you are using.
```

Comment:
change the `subjects.namespace` to the one in which you're deploying the Cluster Operator.